src/ecs/entities/DroppedItem.cpp

```diff
@@ -6,22 +6,22 @@
 */
 
 #include "DroppedItem.hpp"
-#include "../Components/HealthComponent.hpp"
-#include "../Components/DisplayableComponent.hpp"
-#include "../Components/ColliderComponent.hpp"
-#include "../Components/PositionComponent.hpp"
+#include "../components/HealthComponent.hpp"
+#include "../components/DisplayableComponent.hpp"
+#include "../components/ColliderComponent.hpp"
+#include "../components/PositionComponent.hpp"
 #include "../../config.hpp"
-#include "../Components/PowerUpComponent.hpp"
-#include "../Components/PickableComponent.hpp"
-#include "../Components/PowerUpPickedComponent.hpp"
+#include "../components/PowerUpComponent.hpp"
+#include "../components/PickableComponent.hpp"
+#include "../components/PowerUpPickedComponent.hpp"
 
-ECS::DroppedItem::DroppedItem(unsigned id, const Ressources &ressources) :
+ECS::DroppedItem::DroppedItem(unsigned id, Ressources &ressources) :
```

Comment:
I don't know if the DroppedItem entity is used. If not, i think we can just remove it

Reply:
Or maybe inheritate him for the dropped bonus ? I don't know, let you choose